perceval/rendering/circuit/debug_skin.py

```diff
@@ -33,218 +33,25 @@
 from perceval.components import AComponent, Circuit, Port, Herald, PortLocation,\
     unitary_components as cp,\
     non_unitary_components as nu
-from .abstract_skin import ASkin, ModeStyle
+from .abstract_skin import ModeStyle
+from .phys_skin import PhysSkin
 from .skin_common import bs_convention_color
 
 
-class DebugSkin(ASkin):
+class DebugSkin(PhysSkin):
     def __init__(self, compact_display: bool = False):
-        super().__init__({"stroke": "darkred", "stroke_width": 10},
-                         {"width": 2,
-                          "fill": "lightpink",
-                          "stroke_style": {"stroke": "darkred", "stroke_width": 1}},
-                         compact_display)
+        super().__init__(compact_display)
+        self.style[ModeStyle.PHOTONIC]["stroke_width"] = 8
         self.style[ModeStyle.HERALD] = {"stroke": "yellow", "stroke_width": 1}  # Display ancillary modes in yellow
```

Comment:
Yellow is not a very visible color - it also changed with IDE theme/background color of the screen (particularly bad on light themes). Please change the color.